### PR TITLE
Some zh-CN translation fixes.

### PR DESCRIPTION
**Pull request by yhd12138**
Note: Frequency units (GHz, MHz) and file size units (KB, MB, GB etc.) have been changed to English due to preferences of Chinese people. If this is not allowed, I will change them back to zh-CN.

### CODE REVIEW

File: lang/lang.txt [hunk 1]
```diff
@@ -1854,7 +1854,7 @@ pt=Latência do CAS da memória principal atual.
 it=Latenza CAS della memoria principale corrente.
 nl=Huidig hoofdgeheugen CAS Latency.
 da=Aktuel hovedhukommelse CAS Latency.
-zh-CN=当前主存 CAS 延迟。
+zh-CN=当前主内存的 CAS 延迟。; CL
```
Comment (malikm) [inline lang/lang.txt:1857]: Is the text after ; intended as comment? If yes, please remove it.

File: lang/lang.txt [hunk 2]
```diff
@@ -1881,7 +1881,7 @@ pt=Atraso RAS-to-CAS da memória principal atual.
 it=Ritardo RAS-to-CAS della memoria principale corrente.
 nl=Huidig hoofdgeheugen RAS-naar-CAS-vertraging.
 da=Aktuel hovedhukommelse RAS-til-CAS-forsinkelse.
-zh-CN=当前主存储器 RAS-to-CAS 延迟。
+zh-CN=当前主内存的 RAS-to-CAS 延迟。; (tRCD)
```
Comment (malikm) [inline lang/lang.txt:1884]: Same as above - comment?

File: lang/lang.txt [hunk 3]
```diff
@@ -1908,7 +1908,7 @@ pt=Tempo de pré-carga da linha da memória principal atual.
 it=Tempo di precarica riga memoria principale corrente.
 nl=Huidig hoofdgeheugen Rij voorlaadtijd.
 da=Aktuel hovedhukommelse Række-foropladningstid.
-zh-CN=当前主存储器行预充电时间。
+zh-CN=当前主内存的行预充电时间。; (tRP)
```
Comment (malikm) [inline lang/lang.txt:1911]: Same as above - comment?

File: lang/lang.txt [hunk 4]
```diff
@@ -1935,7 +1935,7 @@ pt=Memória principal atual ativa até o tempo de pré-carga.
 it=Memoria principale corrente Attivo al tempo di precarica.
 nl=Huidig hoofdgeheugen Actief tot voorlaadtijd.
 da=Aktuel hovedhukommelse Aktiv til Precharge Time.
-zh-CN=当前主存储器活动到预充电时间。
+zh-CN=当前主内存的活动到预充电时间。; (tRAS)
```
Comment (malikm) [inline lang/lang.txt:1938]: Same as above - comment?

File: lang/lang.txt [hunk 5]
```diff
@@ -1962,7 +1962,7 @@ pt=Memória principal atual de Ativo para Tempo de Ativo / Atualizar.
 it=Memoria principale corrente Attivo ad Attivo/Tempo di aggiornamento.
 nl=Huidig hoofdgeheugen Actief tot Actief/Vernieuwingstijd.
 da=Aktuel hovedhukommelse Aktiv til Aktiv/Opdater tid.
-zh-CN=当前主内存活动到活动/刷新时间。
+zh-CN=当前主内存的活动到活动/刷新时间。; tRC
```
Comment (malikm) [inline lang/lang.txt:1965]: Same as above - comment?

File: lang/lang.txt [hunk 6]
```diff
@@ -1989,7 +1989,7 @@ pt=Tempo de recuperação de atualização da memória principal atual.
 it=Memoria principale corrente Aggiorna tempo di ripristino.
 nl=Huidig hoofdgeheugen Vernieuwen Hersteltijd.
 da=Aktuel hovedhukommelse Opdater gendannelsestid.
-zh-CN=当前主内存刷新恢复时间。
+zh-CN=当前主内存的刷新恢复时间。; tRFC
```
Comment (malikm) [inline lang/lang.txt:1992]: Same as above - comment?

File: lang/lang.txt [hunk 7]
```diff
@@ -2016,7 +2016,7 @@ pt=Taxa de comando da memória principal atual.
 it=Command Rate corrente della memoria principale.
 nl=Huidige hoofdgeheugen Commandosnelheid.
 da=Aktuel hovedhukommelse Kommandohastighed.
-zh-CN=当前主存储器命令速率。
+zh-CN=当前主内存的命令速率。; CR
```
Comment (malikm) [inline lang/lang.txt:2019]: Same as above - comment?

**Review by malikm — COMMENTED** [review]
Please review items meant as comments?

File: lang/lang.txt (hunk 1 above)
Comment (yhd12138) [reply lang/lang.txt:1857]: Yes, they are comments that I forgot to remove before. Now they have been removed.

**Pull request merged**